src/cs50.c

```diff
@@ -57,7 +57,6 @@
 #undef get_int
 #undef get_long_long
 #undef get_string
-#define null (&(struct prompt) {0})
```

Comment:
Is there a clean way we could re-`define` these (for `make hack`)? Or need we relocate the `#define`s from `cs50.h` to a third file that gets `#include`'d by `cs50.h` and also concatenated on to the end of the output of `make hack`?